bower.json

```diff
@@ -1,14 +0,0 @@
-{
-  "name": "Silencio",
-  "version": "1.5.0",
-  "homepage": "https://github.com/viastudio/Silencio",
-  "description": "A custom starter theme for VIA projects",
-  "dependencies": {
-    "jquery": "~2.1.4",
-    "bootstrap": "~3.3.5",
-    "fitvids": "~1.1.0",
-    "respond": "~1.4.2",
```

Comment:
Whoops, I missed telling you about one library. We also need to switch over `respond`. Look for the line `var respond = gulp.src('res/components/respond/dest/respond.src.js')` in **gulpfile.js**